main.py

```diff
@@ -2,6 +2,41 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.bulk_create(
+        [Genre(name="Western"),
+         Genre(name="Action"),
+         Genre(name="Dramma")]
+    )
+    Actor.objects.bulk_create(
+        [
+            Actor(first_name="George", last_name="Klooney"),
```

Comment:
There is a typo in the actor's name 'George Klooney'. It should be 'George Clooney'. This is corrected later in the code, but it's better to avoid introducing incorrect data initially.